uint64 frequency
float64 sampleRate
uint32 lnaGain
uint32 vgaGain
uint32 txvgaGain
int8 ampEnable
int8 antennaEnable
